src/main/php/scriptlet/HttpScriptletRequest.class.php

```diff
@@ -281,6 +281,14 @@ public function getSessionId() {
    * @return string
    */
   protected function importParam($value) {
+
+    // Handle file uploads - these are represented as arrays containg elements for each
+    // file being uploaded
```

Comment:
I'm not sure file uploads are the only parameters which can be values. What about if I invoke a website with `?order[]=size&order[]=name`?